README.md

```diff
@@ -10,18 +12,41 @@ This is the EnergyPlus Development Repository.  EnergyPlus™ is a whole buildin
  - Many users (and developers) of EnergyPlus are active on [Unmet Hours](https://unmethours.com/), so that's a great place to start if you have a question about EnergyPlus or building simulation.
  - For more in-depth, developer-driven support, please utilize the [EnergyPlus Helpdesk](http://energyplus.helpserve.com/).
 
+## Testing
```

Comment:
New testing section with github action badges -- could definitely add more if we wanted to include all of them.